ledger/ledgercore/statedelta.go

```diff
@@ -68,6 +67,12 @@ type Txlease struct {
 	Lease  [32]byte
 }
 
+// IncludedTransactions defines the transactions included in a block, their index and last valid round.
+type IncludedTransactions struct {
+	LastValid        basics.Round
+	TransactionIndex uint64 // the index of the transaction in the block
```

Comment:
Indexer refers to this as "intra", as in "intra round offset". If you like that, it would be great to use the term here also.

Reply:
done